backends/kv/inferschema.go

```diff
@@ -131,16 +141,30 @@ func schemaFromKeys(keyField string, rowSet []map[string]interface{}) (v3ioutils
 				}
 				columnCanBeSortingKey[attrName] = columnCanBeSortingKey[attrName] && attrValueAsString == sortingKeyValue
 			}
+			if hashedPrimaryKeyValue != "" {
+				if _, ok = columnCanBeHashedPrimeryKey[attrName]; !ok {
+					columnCanBeHashedPrimeryKey[attrName] = true
+				}
+				columnCanBeHashedPrimeryKey[attrName] = columnCanBeHashedPrimeryKey[attrName] && attrValueAsString == hashedPrimaryKeyValue
```

Comment:
If it's false, set it to true, then use it in a condition? Why not replace these 4 lines with:
```
columnCanBeHashedPrimeryKey[attrName] = attrValueAsString == hashedPrimaryKeyValue
```

Reply:
**if it does not exists initialize it with true**, you want to have an accumulative indication not just for the current row.

Reply:
Got it. Would be simpler if default was false :)